w3d_translator/read_xml.py

```diff
@@ -39,12 +47,51 @@ def read_xml(file):
     for tag in root.find("ParticleActionRoot") or []:
         pass  # Dict of ParticleActionList by name
 
-    # Globals
-    g = root.find("Global")
-    # TODO: Build <CameraPos> and <CaveCameraPos> (12)
-    # story["camera_pos"] = parse_camera_pos(g.find("CameraPos"))
-    # story["cave_camera_pos"] = parse_cave_camera_pos(g.find("CaveCameraPos"))
-    story["background"] = Color(g.find("Background"))
-    story["wand_navigation"] = WandNavigation(g.find("WandNavigation"))
-
     return story
+
+
+def parse_camera(xml: ET.Element) -> dict:
+    return {
+        "far_clip": float(xml.attrib["far-clip"]),
+        "placement": parse_placement(xml.find("Placement")),
+    }
+
+
+def parse_placement(xml: ET.Element) -> dict:
+    object = {
```

Comment:
I'd change this name to `game_object` or something, just because the word `object` is way too overloaded. Not required, but might be nice

Reply:
I'll go with `placement` for now. I totally did just name it object as the defacto I just need a variable name here haha